sale_order_line_tree_view/views/sale_order_line_views.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_sale_order_line_tree">
+        <field name="name">sale.order.line.tree</field>
+        <field name="model">sale.order.line</field>
+        <field name="arch" type="xml">
+            <tree string="Sales Orders Lines">
+                <field name="order_id"/>
+                <field name="order_partner_id"/>
+                <field name="categ_id"/>
+                <field name="name"/>
+                <field name="lot_id"/>
+                <field name="salesman_id"/>
+                <field name="product_uom_qty"/>
+                <field name="qty_delivered"/>
+                <field name="qty_invoiced"/>
+                <field name="qty_to_invoice"/>
+                <field name="price_subtotal" sum="Total" widget="monetary"/>
+                <field name="state"/>
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_sale_order_line_search">
+        <field name="name">sale.order.line.search</field>
+        <field name="model">sale.order.line</field>
+        <field name="arch" type="xml">
+            <search string="Search options">
+                <filter string="To Be Delivered"
+                        domain="[('to_be_delivered','=',True)]"/>
+                <filter string="Sales"
+                        domain="[('state','in',('sale','done'))]"/>
+                <separator/>
+                <field name="categ_id"/>
+                <field name="order_partner_id"/>
+                <group expand="0" string="Group By">
+                    <filter string="Expected Date"
+                            domain="[]"
+                            context="{'group_by':'expected_date'}"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="action_sale_order_line" model="ir.actions.act_window">
+        <field name="name">Sales Orders Lines</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">sale.order.line</field>
+        <field name="view_id" ref="view_sale_order_line_tree"/>
+        <field name="search_view_id" ref="view_sale_order_line_search"/>
+        <field name="domain">[('is_delivery','!=',True)]</field>
+        <field name="context">{}</field>
+    </record>
+
+    <menuitem action="action_sale_order_line"
+              id="menu_sale_order_line"
+              parent="sales_team.menu_sales"
+              sequence="13"
+              groups="sales_team.group_sale_salesman,sales_team.group_sale_manager"/>
+
+    <menuitem id="sale.menu_product_template_action"
+              action="product.product_template_action"
+              parent="sales_team.menu_sales"
```

Comment:
I was struggling a bit, as I was the menu "Sales Orders Lines" stays below "Sales Orders", while the sequence of "Sales Orders" and "Product" are 12 and 13 respectively.
If 13 is assigned to the sequence of "Sales Orders Line", it will be assigned below the menu "Product"